docs/articles/documentation/guides/continuous-integration/github-actions-and-browserstack.md

```diff
@@ -49,6 +51,28 @@ jobs:
 
 This job runs on a GitHub-hosted virtual machine with the latest Windows version. `test` is the [job ID](https://docs.github.com/en/actions/reference/workflow-syntax-for-github-actions#jobsjob_id) that must be unique to the `jobs` object.
 
+> You can use a GitHub-hosted virtual machine with *any base OS* to run tests. For the sake of simplicity, all examples listed here run on `windows-latest`.
+
+Alternatively, you can [*host your own runners*](https://docs.github.com/en/actions/hosting-your-own-runners/about-self-hosted-runners) for the job. This might be beneficial if you need more precise control over the environment.
```

Comment:
> This might be beneficial if you need more precise control over the environment.

We'll save many words if we write:

`This gives you more precise control over the environment.`